### ISSUE TITLE
Windows ARM64 support (blocked on Go 1.17)

### ISSUE THREAD

**Issue by eamodio** [evanw/esbuild#995]
On a Surface Pro X (a Windows ARM64 device), I getting the following when trying to `yarn` install esbuild.

```
Unsupported platform: win32 arm64 LE
```

Can this platform be supported?

Thanks!

**Comment by evanw** [comment]
Unfortunately it looks like Go doesn't support this specific compilation target. I tried to build this platform but the Go compiler gives this error:

```
cmd/go: unsupported GOOS/GOARCH pair windows/arm64
```

I think this means I am unable to offer native support for your machine.

Caveat: I'm unfamiliar with both Windows and ARM. However, one idea I had is that perhaps a 32-bit ARM executable might work on your machine. It looks like Go does support that compilation target. Can you try these commands on your own to see if they work for you (using Windows Command Prompt syntax)?

```
git clone https://github.com/evanw/esbuild.git
cd esbuild
set GOOS=windows
set GOARCH=arm
go build -o esbuild.exe .\cmd\esbuild
.\esbuild.exe --version
```

What output does that give you?

**Comment by eamodio** [comment]
It looks like there is progress on Go supporting Windows/ARM64: https://github.com/golang/go/issues/36439#issuecomment-786102642

I'll try out the above tomorrow and let you know the results.

**Comment by evanw** [comment]
Closing this issue as this is currently impossible given Go's lack of a target for this. We should reopen this issue or create another issue if this ever becomes possible in the future.

**Comment by evanw** [comment]
Reopening now that https://github.com/golang/go/issues/36439 has been resolved. It sounds like support for Windows ARM64 is in for Go 1.17, which I believe is scheduled for August 2021.

**Comment by evanw** [comment]
I gave this a shot now that Go 1.17.0 has been released. If you `npm install esbuild-windows-arm64` on a Windows ARM64 machine, does `npx esbuild --version` work now?

**Comment by eamodio** [comment]
Yup, that worked! Thanks!

**Comment by eamodio** [comment]
@evanw once a new version of esbuild is released with this included -- Windows ARM64 support should be seamless correct?

**Comment by evanw** [comment]
> @evanw once a new version of esbuild is released with this included -- Windows ARM64 support should be seamless correct?

Yes, that's correct. After the next esbuild version is published, you should be able to just `npm install esbuild` on Windows ARM64. I'm going to publish a new version sometime today.
